src/shared/modules/commands/useDb.test.js

```diff
@@ -1,76 +0,0 @@
-/*
```

Comment:
Why was this test removed?

Reply:
Removed it since it's skipped and has been skipped for a long time. Should i put it back in?

Reply:
Looking at history, it was skipped with the comment "Ignoring one test for now. Need to mock `driver.supportsMultiDb()`". I agree that there no reason to keep skipping it, either remove or add that mock is better than skipping